ansible/roles/ansible-horizon-plugin/tasks/collect_compress.yml

```diff
@@ -54,5 +76,3 @@
 - debug:
     msg: "\033[33m WARNING: Unable to locate local_settings.d on system. Trilio quota feature on dashboard may not work. \033[0m"
     verbosity: "{{ verbosity_level }}"
-  when: os_local_settings_path == ""
```

Comment:
Is this intentional change?

Reply:
no, I need to revert it. mistake it was.
will correct it.